src/chrome_trace/chrome_trace.mli

```diff
@@ -18,6 +18,54 @@ module Json : sig
     | Object of (string * t) list
 end
 
+module Event : sig
+  type t
+
+  module Timestamp : sig
+    type t
+
+    val of_float_seconds : float -> t
+
+    val now : unit -> t
+  end
+
+  module Id : sig
+    type t =
+      | Int of int
+      | String of string
+  end
+
+  type common
+
+  val common :
```

Comment:
`Event.common` sounds like you're creating a common event, or something. Three ideas, not sure if any of them is any good or worth making a change for:

1. Rename `common` -> `common_fields`. (pretty self-explanatory)

2. Stop exposing the `common` type altogether, do this instead:
```
type 'a with_common_args = 
  ?tts:Timestamp.t
  -> ...
  -> 'a

val counter : (?id:Id.t -> args -> t) with_common_args.t
```
(a common idiom in Jane Street, even if not the most beautiful one)

3. Have a single create function like this:
```
type details =
  | Counter of ...
  | ...
val create : ?tts:Timestamp.t -> ... -> details -> t
```

Reply:
I'll go for 2.

Reply:
Actually, 2 makes `Event.set_ts` awkward. Let me try 3.

Reply:
In the end, I went for 3 as its the least hassle.